ambari-server/src/main/resources/Ambari-DDL-Derby-CREATE.sql

```diff
@@ -613,11 +613,11 @@ CREATE TABLE blueprint_mpack_instance(
 
 CREATE TABLE blueprint_service (
   id BIGINT NOT NULL,
-  mpack_ref_id BIGINT NOT NULL,
+  mpack_instance_id BIGINT NOT NULL,
```

Comment:
Doesn't this require a change in `BlueprintServiceEntity`, too?

https://github.com/apache/ambari/blob/80d840e5d50614a0f148a650c9bfc7d21363a218/ambari-server/src/main/java/org/apache/ambari/server/orm/entities/BlueprintServiceEntity.java#L50

Reply:
Fixed. Thanks.